command/certificate/verify.go

```diff
@@ -164,6 +177,30 @@ func verifyAction(ctx *cli.Context) error {
 		}
 	}
 
+	if expire {
+
+		NowTillEndOfCert := time.Until(cert.NotAfter)
+		totalLifeTimeOfCert := cert.NotAfter.Sub(cert.NotBefore)
```

Comment:
I would check first if `time.Now().Before(cert.NotBefore)` because if so, then the cert is not valid yet, and that's a different message than any of the ones you have below. At the same time I would check if `time.Now().After(cert.NotAfter)` just to rule out both of those special cases. Then we know that the results should be within our bounds.

I would also do `(1 - remainder/totalValidity) = `